third_party/parquet/parquet_types.h

```diff
@@ -92,7 +92,8 @@ struct Encoding {
     DELTA_BINARY_PACKED = 5,
     DELTA_LENGTH_BYTE_ARRAY = 6,
     DELTA_BYTE_ARRAY = 7,
-    RLE_DICTIONARY = 8
+    RLE_DICTIONARY = 8,
+    BYTE_STREAM_SPLIT = 9,
```

Comment:
This file has a comment saying it's autogenerated and not to edit it, but from what I can tell this is autogenerated in the Arrow repository and was copied into DuckDB, so is fine to edit. Is that correct?